cli/state.rs

```diff
@@ -195,9 +196,14 @@ impl ThreadSafeState {
       Some(file_name) => {
         let base_url = match &main_module {
           Some(module_specifier) => module_specifier.clone(),
-          None => unreachable!(),
+          // Dummy child of the CWD.
+          None => ModuleSpecifier::resolve_url_or_path("_").unwrap(),
         };
-        let import_map = ImportMap::load(&base_url.to_string(), file_name)?;
+        let import_map = ImportMap::load(&base_url.to_string(), file_name)
+          .unwrap_or_else(|err| {
+            eprintln!("{}", err.to_string());
+            process::exit(1);
```

Comment:
This seems wrong. Why doesn't raising the error work?

Can you add tests?

Reply:
It's being unwrapped in main.rs. You're right, I'll exit there instead.